cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -206,6 +206,30 @@ def _create_device_spec_from_template(template_name: str) -> v2.device_pb2.Devic
     return device_spec
 
 
+def create_device_from_processor_id(
+    processor_id: str,
+    gate_sets: Optional[Iterable[serializable_gate_set.SerializableGateSet]] = None,
```

Comment:
I recommend omitting this arg and always set the gateset to FSIM_GATESET. The new Device implementation (`cirq_google.GridDevice`) will not have gatesets as a parameter to `from_proto`.

Reply:
Done.